docs/examples/README.md

```diff
@@ -0,0 +1,8 @@
+# Doc Examples
+
+In this directory you can find:
+
+- `apm`: A practical example of integrating elasticsearch-ruby and [Elastic APM](https://www.elastic.co/apm) with the [Ruby APM Agent](https://github.com/elastic/apm-agent-ruby). Check [the README](https://github.com/elastic/elasticsearch-ruby/tree/master/docs/examples/apm/README.md#observability-example) for more information.
+- `guide`: Contains the generated Ruby code examples for the [Elasticsearch Reference](https://www.elastic.co/guide/en/elasticsearch/reference/master/index.html).
```

Comment:
We use the `current` version in the root README.md. I'd use that here for consistency.
```suggestion
- `guide`: Contains the generated Ruby code examples for the [Elasticsearch Reference](https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html).
```